internal/beater/api/mux.go

```diff
@@ -103,18 +104,26 @@ func NewMux(
 		intakeSemaphore:  make(chan struct{}, beaterConfig.MaxConcurrentDecoders),
 	}
 
+	zapLogger := zap.New(logger.Core(), zap.WithCaller(true))
+	builder.intakeProcessor = elasticapm.NewProcessor(elasticapm.Config{
```

Comment:
Note to reviewer: while moving processor/stream to apm-data, I simplified the API so that the processor will handle either "m" or "metadata" for the first line, like we were already doing for event type handling ("x" vs. "transaction"). This means that we can use the same processor for both v2 and RUMv3.